### ISSUE TITLE
RTL for docs translation

### ISSUE THREAD

**Issue by bluemix** [hexojs/hexo#2677]
Hello dears,
How can I add an Arabic translation to the 
docs if it doesn't supports RTL?
Or can I modify the navy theme to supports RTL?

**Comment by tea3** [comment]
@bluemix 
I do not know about navy-theme, but I think that I can customize it. If you change the theme's CSS file as follows, it may be displayed normally.

```
direction: rtl;
unicode-bidi: embed;
text-align: right;
float: left;
clear: right;
```

**Comment by NoahDragon** [comment]
@bluemix I saw you have submitted two plugins, which is the recommended way to add a new feature to Hexo.

Correct me if I'm wrong, are you asking how to update the Hexo document?

**Comment by bluemix** [comment]
@tea3 actually I got problems with the `unicode-bidi: embed;` and when forcing `direction: rtl;`. because we have a mixed LTR (e.g., code) and RTL (e.g., explanations) texts. I've written two tag plug-ins what force the layout director for [LTR](https://github.com/bluemix/hexo-tag-ltr) or [RTL](https://github.com/bluemix/hexo-tag-rtl). I mean it is ok to use tag plugs in the site, right?
The other question is that what is the possibility of using a unicode-bidi algorithm, like [this](https://github.com/bbc/unicode-bidirectional), so that the above RTL<->LTR fixes will not be needed.

**Comment by bluemix** [comment]
@NoahDragon thanks for the merge 😇
Ah, I am sorry, but this issue was supposed to be about the Hexo site.

I was asking If we can add the Arabic language to the Hexo site, like 
having an additional Arabic language in this screenshot
<img width="160" alt="screen shot 2017-07-31 at 17 19 30" src="https://user-images.githubusercontent.com/3332274/28781980-869f6e70-7614-11e7-90d4-f87cb77895d9.png">

**Comment by NoahDragon** [comment]
@bluemix There is no Arabic translation so far. Would you like to help us translate doc/api to Arabic? If so, that's would be great. 😄 

Here is an under development translation for Portuguese. The Arabic translation file structure would be same as this: https://github.com/lucianobarauna/site/tree/new_translate_ptbr/source/pt-br

**Comment by bluemix** [comment]
@NoahDragon yes, I'd like to translate the docs to Arabic. the main problem is that the Arabic language is written from right-to-left (RTL languages) 😁
So I was discussing with @tea3 about this problem :)

**Comment by NoahDragon** [comment]
@bluemix That would be awesome. 👍 

To make text direct from right to left, I think the plugin you just created would be perfect. You could add it to the package dependency and use it anywhere it needed.

**Comment by bluemix** [comment]
ok, I'll start the Arabic translation and I hope tings go well.
thanks dears :)
